src/test/suite/stubs.ts

```diff
@@ -317,7 +308,7 @@ class LanguageServerControllerStub {
     return;
   }
 
-  executeAll(/* codeToEvaluate: string */): Promise<ShellExecuteAllResult> {
+  evalauate(/* codeToEvaluate: string */): Promise<ShellEvaluateResult> {
```

Comment:
```suggestion
  evaluate(/* codeToEvaluate: string */): Promise<ShellEvaluateResult> {
```

(similarly elsewhere)